src/openrct2-ui/windows/Footpath.cpp

```diff
@@ -613,14 +613,7 @@ static void window_footpath_paint(rct_window* w, rct_drawpixelinfo* dpi)
 
         int32_t selectedPath = gFootpathSelectedId;
```

Comment:
int32_t selectedPath = `(gFootpathSelectedType << 7) +  gFootpathSelectedId;` otherwise it will only select the regular path.

Reply:
I think that should be `<< 5` or `* 32`. There are also other approaches, though, since I haven't done the split 100%.